webplugin/css/app/mck-sidebox-1.0.css

```diff
@@ -5533,6 +5548,25 @@ div#mck-rated:before{
 		opacity: 1;
 	}
 }
+
+.mck-mic-btn {
+	display: inline-block;
+	float: left;
+	margin-top: 12px;
+}
+
+.mck-mike-btn {
```

Comment:
mike -> mic :) existing code issue

Reply:
Didn't made any changes in the previous PR, that's why WIP.